addons/pandora/ui/editor/property_settings_editor/property_settings_editor.gd

```diff
@@ -13,6 +13,15 @@ const EntityPicker = preload("res://addons/pandora/ui/components/entity_picker/e
 var _property:PandoraProperty
 var _default_settings:Dictionary
 
+var allowed_nested_types: Dictionary = {
```

Comment:
This potentially can be improved by adding a new boolean method to `PandoraPropertyType` that determines if a type can be nested or not. All we then have to do is to add a new method to [type.gd](https://github.com/bitbrain/pandora/blob/godot-4.x/addons/pandora/model/type.gd) like this:
```gdscript
static func get_all_types() -> Array[PandoraPropertyType]
```
That can be used to find and lookup all types where the `allow_nesting()` may be true.

Reply:
I like that! Will update the PR